File: onetimepadgtk/ErrorDialog.cs
using System;
using Gtk;

namespace onetimepadgtk
{
    public static class ErrorDialog
    {
        public static void ShowAlert(Window parent, string text)
        {
            MessageDialog dialog = new MessageDialog(
                parent,
                0,
                Gtk.MessageType.Error,
                ButtonsType.Ok,
                text
            );
            dialog.Run();
            dialog.Destroy();
        }
    }
}

File: UI/ErrorDialog.py
import gi

gi.require_version("Gtk", "3.0")
from gi.repository import Gtk


def ShowAlert(parent, text):
    dialog = Gtk.MessageDialog(
        transient_for=parent,
        flags=0,
        message_type=Gtk.MessageType.ERROR,
        buttons=Gtk.ButtonsType.OK,
        text="ERROR",
    )
    dialog.format_secondary_text(
        text
    )
    dialog.run()

    dialog.destroy()
